doc/faq.rst

```diff
@@ -1,147 +1,177 @@
+  config HV_RAM_SIZE
+    hex "Size of the RAM region used by the hypervisor"
+    default 0x07800000 if PLATFORM_SBL
+    default 0x0f000000 if PLATFORM_UEFI
+
+How to modify the default display output for a UOS?
+***************************************************
+
+Apollo Lake HW has three pipes and each pipe can have three or four planes which
+help to display the overlay video. The hardware can support up to 3 monitors
+simultaneously. Some parameters are available to control how display monitors
+are assigned between the SOS and UOS(s), simplifying the assignment policy and
+providing configuration flexibility for the pipes and planes for various IoT
+scenarios. This is known as the **plane restriction** feature.
+
+* ``i915.avail_planes_per_pipe``: for controlling how planes are assigned to the
+  pipes
+* ``i915.domain_plane_owners``: for controlling which domain (VM) will have
+  access to which plane
+
+Refer to :ref:`GVT-g-kernel-options` for detailed parameter descriptions.
+
+In the default configuration, pipe A is assigned to the SOS and pipes B and C
+are assigned to the UOS, as described by these parameters:
+
+* SOS::
+
+    i915.avail_planes_per_pipe=0x01010F
+    i915.domain_plane_owners=0x011111110000
+
+* UOS::
+
+    i915.avail_planes_per_pipe=0x0070F00
+
+To assign pipes A and B to the UOS, while pipe C is assigned to the SOS, use
+these parameters:
+
+* SOS::
+
+    i915.avail_planes_per_pipe=0x070101
+    i915.domain_plane_owners=0x000011111111
+
+* UOS::
+
+    i915.avail_planes_per_pipe=0x000F0F
+
+.. note::
+
+   The careful reader may have noticed that in all examples given above, the SOS
+   always has at least one plane per pipe. This is intentional, and the driver
+   will enforce this if the parameters do not do this.
+
+Why does ACRN need to know how much RAM the system has?
+*******************************************************
+
+Configuring ACRN at compile time with the system RAM size is a tradeoff between
+flexibility and functional safety certification. For server virtualization, one
+binary is typically used for all platforms with flexible configuration options
+given at run time. But, for IoT applications, the image is typically configured
+and built for a particular product platform and optimized for that product.
+
+Important features for ACRN include Functional Safety (FuSa) and real-time
+behavior. FuSa requires a static allocation policy to avoid the potential of
+dynamic allocation failures. Real-time applications similarly benefit from
+static memory allocation. This is why ACRN removed all ``malloc()``-type code,
+and why it needs to pre-identify the size of all buffers and structures used in
+the Virtual Memory Manager. For this reason, knowing the available RAM size at
+compile time is necessary to statically allocate memory usage.
+
+How to workaround compilation issue on Fedora 29?
+*************************************************
+
+Please refer to following information for workaround::

```

Comment:
I'll let @dbkinder recommend the exact phrasing but we should:
1. Give a short summary of the issue, e.g.: "There is a known issue when attempting to build ACRN on Fedora 29. This is due to the way `gnu-efi` is packaged up in Fedora.
2. Add a reference to the issue on Github some people can see the whole thread/discussion/history

Reply:
Interesting that you changed some uses of the ``ARCH`` variable to be hard coded as ``x64``, but not all of them?  In any case, yes, this needs an intro as @gvancuts commented.  Something like:

```
How to build ACRN on Fedora 29
******************************

There is a known issue when attempting to build ACRN on Fedora 29
because of how ``gnu-efi`` is packaged in this Fedora release.
(See the `ACRN GitHub issue
<https://github.com/projectacrn/acrn-hypervisor/issues/2457>`_ 
for more information.)  The following patch to ``/efi-stub/Makefile``
fixes the problem on Fedora 29 development systems (but should
not be used on other Linux distros)::
```

Reply:
> Interesting that you changed some uses of the `ARCH` variable to be hard coded as `x64`, but not all of them?

I wanted to fix the paths and names of utilities. The only other `ARCH` that's left is part of the `CFLAGS` and used as a macro. I don't actually know what that does, if anything at all, so I did not touch it :-)